worker/src/index.ts

```diff
@@ -51,6 +52,18 @@ function createTarballWorkerPool (): WorkerPool {
   return workerPool
 }
 
+function calcMaxWorkers () {
+  if (process.env.PNPM_WORKERS) {
+    const idleCPUs = Math.abs(process.env.PNPM_WORKERS ? parseInt(process.env.PNPM_WORKERS) : 0)
```

Comment:
```suggestion
    const idleCPUs = Math.abs(parseInt(process.env.PNPM_WORKERS))
```
The repeated judgment of `process.env.PNPM_WORKERS` here seems unnecessary.